### ISSUE TITLE
Output of time units not CF compliant

### ISSUE THREAD

**Issue by atleanalytt** [metno/fimex#37]
The output of a conversion from GRIB to NetCDF is on the format "seconds since 1970-01-01 00:00:00 +0000". This is not compliant with the [Climate and Forecast Metadata Conventions (CF)](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html,) which are using [UDUNITS](https://www.unidata.ucar.edu/software/udunits/). A correct output would read "seconds since 1970-01-01 00:00:00 +00:00", i.e. a colon in the timezone definition.

**Comment by alexander-buerger-met-no** [comment]
Hmm, I do not understand where this comes from. There are lots of matches when searching for `+00:00` but basically none when searching for `0000`:

```
$ grep -rn "seconds since.*0000" *
modules/F90/fimex.f90:484:  !! @param unit optional time-unite, defaults to 'seconds since 1970-01-01 +0000'
modules/F90/fimex.f90:494:    myUnit = "seconds since 1970-01-01 00:00:00 +0000";
share/etc/cdmGribReaderConfigEC_MAD.xml:28:            <attribute name="units" value="seconds since 2014-03-05 00:00:00 +0000" type="string"/>
Binary file … matches
```

Do you have any suggestion where the unit without timezone colon might come from?

**Comment by heikoklein** [comment]
Furthermore, udunits allows both +0000 and +00:00 for timezones:
```udunits2
You have: seconds since 2014-03-05 00:00:00 +0000
You want: seconds since 2014-03-05 00:00:00 +01:00
    1 seconds since 2014-03-05 00:00:00 +0000 = 3601 (seconds since 2014-03-05 00:00:00 +01:00)
    x/(seconds since 2014-03-05 00:00:00 +01:00) = (x/(seconds since 2014-03-05 00:00:00 +0000)) + 3600
You have: seconds since 2014-03-05 00:00:00 +0100    
You want: seconds since 2014-03-05 00:00:00 +0000
    1 seconds since 2014-03-05 00:00:00 +0100 = -3599 (seconds since 2014-03-05 00:00:00 +0000)
    x/(seconds since 2014-03-05 00:00:00 +0000) = (x/(seconds since 2014-03-05 00:00:00 +0100)) - 3600
```

**Comment by atleanalytt** [comment]
I stand corrected on both of your comments. I had not investigated closely enough, sorry about that. I am using a library that is using CF time as standard, and it would not accept the +0000-string, so I assumed this was not accepted by udunits. It's the library that I'm using that's even more restrictive. 
As for the output, I found this is a setting in the config files used, so it's not really an issue. I jumped the gun. Props for quick response!
Happy to close.
